docs/source/column.rst

```diff
@@ -96,42 +98,44 @@ StringColumn class reference
 .. autosummary::
     :toctree: generated
     :nosignatures:
+    :template: class.rst
+
+    istring_column.StringMethods.length
```

Comment:
as a follow-up, maybe we should rename `istring_column.py` to `string_column.py` to get better doc :) 

(btw, how does Pandas render doc or implement `str_col.str.XXX`...)

Reply:
is this how pandas implemented it? https://github.com/pandas-dev/pandas/blob/main/pandas/core/series.py#L5851